cluster/gce/gci/configure-helper.sh

```diff
@@ -980,6 +1010,7 @@ function start-kube-controller-manager {
   if [[ -n "${CA_KEY:-}" ]]; then
     params+=" --cluster-signing-cert-file=/etc/srv/kubernetes/ca.crt"
     params+=" --cluster-signing-key-file=/etc/srv/kubernetes/ca.key"
+    params+=" --insecure-experimental-approve-all-kubelet-csrs-for-group=node-bootstrapper"
```

Comment:
From a quick look at groupapprove.go code it seems like this will approve any cert that has "node-bootstrapper" in the right CSR field. Doesn't that mean that anyone who can touch the CSR API can issue a cert for any node? Turning on a scary-looking option like this deserves some comments.